resources/views/layouts/datagrid/bulks/_map-marker.blade.php

```diff
@@ -1,3 +1,83 @@
+<?php
+/** @var \App\Models\MapMarker $model */
+$campaign = \App\Facades\CampaignLocalization::getCampaign();
+
+$iconOptions = [
+    '' => null,
+    1 => __('maps/markers.icons.marker'),
+    2 => __('maps/markers.icons.question'),
+    3 => __('maps/markers.icons.exclamation'),
+    4 => __('maps/markers.icons.entity'),
+];
+
+$sizeOptions = [
+    '' => null,
+    1 => __('locations.map.points.sizes.tiny'),
+    2 => __('locations.map.points.sizes.small'),
+    3 => __('locations.map.points.sizes.standard'),
+    4 => __('locations.map.points.sizes.large'),
+    5 => __('locations.map.points.sizes.huge'),
+    6 => __('locations.map.points.sizes.custom'),
+];
+
+$typeOptions = [
+    '' => null,
+    0 => __('general.no'),
+    1 => __('general.yes'),
+];
+
+$groups = $model->groupOptions();
+$groups[-1] = __('crud.filters.options.none');
+?>
+<div class="form-group">
+    <label for="icon">{{ __('locations.map.points.fields.icon') }}</label>
+    {!! Form::select('icon', $iconOptions, \App\Facades\FormCopy::field('icon')->string(), ['class' => 'form-control', 'id' => 'icon']) !!}
```

Comment:
We don't need to call FormCopy in this place, since we aren't copying a marker :)